### PR TITLE
Issue#1880 mejs4 create thumbnail

### PR DESCRIPTION
**Pull request by adamjarling**
**Note**: this should be merged **AFTER** #2307

### CODE REVIEW

**Comment by adamjarling** [comment]
This branch is now rebased on top of latest `develop`, and runs as expected locally.

File: app/assets/javascripts/media_player_wrapper/mejs4_create_thumbnail.es6 [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+     * @param {HTMLElement} layers
+     * @param {HTMLElement} media
+     */
+    cleancreateThumbnail (player, controls, layers, media) {
+    },
+
+    // Other optional public methods (all documented according to JSDoc specifications)
+
+    /**
+     * The 'createThumbnailObj' object acts as a namespacer for all Add to Playlist
```
Comment (bkeese) [inline app/assets/javascripts/media_player_wrapper/mejs4_create_thumbnail.es6]: missed Add to Playlist in comment

File: app/assets/javascripts/media_player_wrapper/mejs4_create_thumbnail.es6 [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+        }).fail((error) => {
+          console.log(error);
+        }).always(() => {
+          modalBody.classList.remove('spinner');
+          $(createThumbnailObj.modalEl).find('button').attr({ disabled: false });
+        });
+      },
+
+      /**
+       * Handle control button click to toggle Add Playlist display
```
Comment (bkeese) [inline app/assets/javascripts/media_player_wrapper/mejs4_create_thumbnail.es6]: Here it is again, and the next line

**Comment by adamjarling** [comment]
@bkeese Got 'em.  I just re-pushed with the updates.   And yea there's probably a fair amount of MEJS2 player development happening in parallel which MEJS4 will have to catch up with at some point.   If you notice anything specific and have a few extra minutes, feel free to create a thin issue, tag me, and I'll flesh it out and make sure stuff doesn't slip by.

**Pull request merged**